hillstone.py

```diff
@@ -257,12 +264,14 @@ def wait_network(self):
                 self.server_udp_port = int.from_bytes(res[Payload.SVR_UDP_PORT], byteorder='big')
                 self.ip_ipv4 = ipaddress.IPv4Interface((res[Payload.CLT_PRIV_IPV4], network.prefixlen))
                 self.gateway_ipv4 = ipaddress.IPv4Address(res[Payload.SVR_PRIV_IPV4])
-                self.dns_ipv4 = ipaddress.IPv4Address(res[Payload.DNS_IPV4])
+                self.dns_ipv4 = ipaddress.IPv4Address(self.Int2IP(int.from_bytes(res[Payload.DNS_IPV4],'big')))
                 self.wins_ipv4 = res[Payload.WINS_IPV4]
             elif msg_id == MessageType.SET_ROUTE:
                 self.route_ipv4 = res[Payload.ROUTE_IPV4]
             elif msg_id == MessageType.KEY_DONE:
                 break
+            elif msg_id == MessageType.NONE:
+                continue
```

Comment:
Why is this necessary? The other ids have already been ignored and will go to continue the `while` statement.